localization/translation-cleanup.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
```

Comment:
@codexmarcel Are you OK with me adding a new folder to the root of the repository to contain files unique to the localization process? I'll add a readme.md file too.

Reply:
@codexmarcel To take this whole script a bit further, if we can add a new scratch org definition to the repo that has sampleData disabled, then I can drop the first couple of lines in this bash file that require JQ.

Reply:
Great!. I seem to remember that @davidmreed had some concerns with dropping hasSampleData in the org definitions, but a new scratch org type just for localization would work if we can't modify the existing one.

Also, this pattern (the scratch org def and this script file) will be repeated in the five underlying repositories.